egg/platform_ppc.py

```diff
@@ -61,24 +83,26 @@ def ppc_vec_typel(typ):
     else:
         raise ValueError('Unknown type "{}"'.format(typ))
 
+
 ## Whether or not the half float are emulated
 def emulate_fp16(simd_ext):
     return True
 
-## Emulate 64 bits types (for power7)
-def emulate_64(op, simd_ext, params):
+
+## Emulate 64 bits types (for vsx)
```

Comment:
VSX --> VMX, emulation must be done for VMX VSX supports 64-bits types.
- `__vector signed long`
- `__vector unsigned long`
- `__vector double`